pkg/utils/utils.go

```diff
@@ -679,9 +686,13 @@ type OCPClusterInfo struct {
 }
 
 func getOCPClusterInfo(r *ReconcilerBase) *OCPClusterInfo {
-	config, err := r.GetOperatorConfigMap("console-config", "openshift-console")
+	logger := NewLogger(true) //log in JSON format
```

Comment:
Should be passing the Logger from Reconcile() down to this function so that it uses the logging setting from the kAppNav CR.